test/unit/addons/p5.dom.js

```diff
@@ -1,123 +0,0 @@
-/* global testSketchWithPromise */
```

Comment:
@lmccart can we keep these tests? or were they moved elsewhere and i'm just not seeing them?

Reply:
oops good catch, i'll restore them and move to `test/unit/dom`